src/main/java/fr/insalyon/citi/golo/compiler/JavaBytecodeGenerationGoloIrVisitor.java

```diff
@@ -451,14 +454,26 @@ public void visitThrowStatement(ThrowStatement throwStatement) {
     methodVisitor.visitInsn(ATHROW);
   }
 
-  private void visitInvocationArguments(AbstractInvocation invocation) {
-    for (ExpressionStatement statement : invocation.getArguments()) {
-      statement.accept(this);
+  private List<String> visitInvocationArguments(AbstractInvocation invocation) {
+    List<String> argumentNames = new ArrayList<>();
+    for (ExpressionStatement argument : invocation.getArguments()) {
+      if (invocation instanceof FunctionInvocation && ((FunctionInvocation) invocation).usesNamedArguments()) {
```

Comment:
Indeed...
Fixed
https://github.com/danielpetisme/golo-lang/commit/5a82c9156fad4004468389ba302e8249af0f3951#diff-798c54ba96714a3ddf68f1a3db92b6a3R28

As mentionned, I pulled up the named argument stuff into `AbstractInvocation`
https://github.com/danielpetisme/golo-lang/blob/5a82c9156fad4004468389ba302e8249af0f3951/src/main/java/fr/insalyon/citi/golo/compiler/ir/AbstractInvocation.java